index.html

```diff
@@ -69,14 +69,21 @@
           for (let prac of document.querySelectorAll('div.practice.advisement')) {
             prac.classList.remove('advisement');
             prac.classList.add('principle');
+            const anchor = prac.querySelector('span.practicelab');
+            const summary = document.querySelector(`#bp-summary a.marker.self-link[href="#${anchor.id}"]`);
+            if (!summary) console.error(`No summary from ${anchor.id}`);
```

Comment:
In the console, use 
```suggestion
            if (!summary) console.error(`No summary from `, anchor);
```
and it'll show the element itself, which lets folks find it in the inspector or on the page to help understand what's wrong. Or if another element is more helpful in seeing where the summary ought to go, you could list both that element and the anchor's id.

Reply:
Done!